backend/utils/generateToken.ts

```diff
@@ -1,16 +1,13 @@
 import jwt from 'jsonwebtoken'
+import { config } from '../config'
 
-// TODO: import from config
 let JWT_SECRET: string
-if (process.env.JWT_SECRET) {
-  JWT_SECRET = process.env.JWT_SECRET
+if (config.jwtSecret) {
+  JWT_SECRET = config.jwtSecret
 } else {
```

Comment:
Unnecessary validation, you already did that in the config file.

Reply:
you're right, removed this part!